src/flowTypes.js

```diff
@@ -105,14 +105,25 @@ export type BoxItemPermission = {
 };
 
 export type User = {
-    avatarUrl: string,
-    email: string,
+    avatarUrl: ?string,
+    email: ?string,
     id: string,
     login: string,
     name: string,
     type: 'user'
 };
 
+export type UserCollection = {
+    total_count?: number,
+    entries?: User[],
```

Comment:
There is a mix match in this file, but since we have started to use it in other places, lets stick with Array<Foo> instead of Foo[]

Reply:
Went through and updated it everywhere else in flowTypes.js